commons/src/main/java/org/eclipse/kapua/commons/configuration/ServiceConfigImpl.java

```diff
@@ -32,7 +32,7 @@
 
 @XmlRootElement
 @XmlAccessorType(XmlAccessType.FIELD)
-@XmlType(propOrder = {"pid", "configurations"})
+@XmlType(propOrder = {"pid"})
 @Entity(name = "ServiceConfig")
```

Comment:
This is a fix unreleated to the current changes - the "configurations" field is marked as @XmlTransient - as such, it is illegal to include it in the propOrder configuration